openstack-helm/trilio-backup-targets/values_overrides/nfs.yaml

```diff
@@ -0,0 +1,17 @@
+trilio_backup_target:
+  backup_target_name: 'NFS_BackupTarget'
+  backup_target_type: 'nfs'
+  is_default: True
+  nfs_server: '10.10.0.1'
+  nfs_shares: /home/openstack-helm
+  nfs_options: "nolock,soft,timeo=600,intr,lookupcache=none,nfsvers=3,retrans=10"
+  storage_size: 20Gi
+  storage_class_name: nfs
+
+common:
```

Comment:
You can remove this common block from here. 
All settings listed in here are almost contstant. We keep in values.yaml with default values.